UnityProject/Assets/Scripts/UI/Objects/Medical/genetics/MutationUnlockMiniGame.cs

```diff
@@ -102,7 +97,7 @@ public void TryGenerateEgg()
 	{
 		if (GUI_DNAConsole.DNAConsole.CurrentDNACharge >= GUI_DNAConsole.DNAConsole.RequiredDNASamples)
 		{
-			GUI_DNAConsole.DNAConsole.CurrentDNACharge = 0;
+			GUI_DNAConsole.DNAConsole.CurrentDNACharge =- GUI_DNAConsole.DNAConsole.RequiredDNASamples;
```

Comment:
like codacy asks, was this intentional? Do you want to asign the negative of ``RequiredDNASamples`` ? If so, then separate the minus symbol from the assignation symbol and append it to the variable itself.

Reply:
yeah,  remove the required amount from the pool